gsa/src/web/pages/login/loginpage.js

```diff
@@ -235,11 +241,15 @@ const mapDispatchToProps = (dispatch, {gmp}) => ({
   setIsLoggedIn: value => dispatch(setIsLoggedIn(value)),
 });
 
+const mapStateToProp = (rootState, ownProps) => ({
```

Comment:
For consistency, could you rename this to mapStateToProps?
```suggestion
const mapStateToProps = (rootState, ownProps) => ({
```